eatsmart/locations/wake/management/commands/import_wake.py

```diff
@@ -154,3 +174,39 @@ def save_violations(self, violations):
                 print('New record')
             violation_obj.__dict__.update(**attributes)
             violation_obj.save()
+
+    def inspection_risk_factors(self):
+        factors = (('hold_temp', 1), ('cook_temp', 2), ('contamination', 3),
+                   ('hygeine', 4), ('source', 5))
+        inspections = Inspection.objects.all()
```

Comment:
You can use select_related to make it better on the db, otherwise inspection.violation_set.all() will hit the db once for each iteration through the loop.